lib/cartopy/io/img_tiles.py

```diff
@@ -79,6 +88,21 @@ def fetch_tile(tile):
         img, extent, origin = _merge_tiles(tiles)
         return img, extent, origin
 
+    @property
+    def _cache_dir(self):
+        """Return the name of the cache directory"""
+        return os.path.join(
+            self.cache_path,
+            self.__class__.__name__
+        )
+
+    def _load_cache(self):
+        """Load the cache"""
+        if self.cache_path is not None:
+            if not os.path.exists(self._cache_dir):
+                os.makedirs(self._cache_dir)
```

Comment:
We could indeed. But maybe only in the case where `cache=True`, because when the user explicitly sets a directory path we can suppose he knows what he is doing.